corpus/BOMTest/UTF8.pm

```diff
@@ -1,4 +1,6 @@
-﻿package Heart;
+﻿use utf8;
```

Comment:
Why is this unrelated change included?

Reply:
I've removed this commit but it kinda looks like whatever code is walking this is not taking into account that the package name has a utf8 char in it but `use utf8` is not declared. I'm not sure of the original intention.